tar-artifacts/action.yml

```diff
@@ -0,0 +1,28 @@
+name: 'Tar build artifacts'
+inputs:
+  kbuild-output:
+    description: 'Path to the kernel build output'
+    required: true
+  repo-root:
+    description: "Path to the root of the kernel repository"
+    required: true
+  arch:
+    required: true
+  toolchain:
+    required: true
+outputs:
+  archive-name:
+    description: 'Artifacts tarball name'
+    value: ${{ steps.run-script.outputs.archive-name }}
```

Comment:
toolchain is only needed so we can generate the final archive name. I wonder if we should rather provide the destination file as an input, this way we can give it the name we like from the caller.

It also seems to be what other actions doing something similar would do: https://github.com/marketplace/actions/tar-action

Reply:
Added `archive` input and removed output.

I thought about this, and wasn't sure what's the best API would be. The caller obviously needs to know the archive name, so why don't just allow them to set it.

Also, it seems so far that using env variables to control action-driven scripts is very convenient. We should think about a convention for when to use env variables vs script argument vs action inputs, because right now it's quite chaotic.

Reply:
> because right now it's quite chaotic.

could not agree more :D